.gitlab-ci-check-apidocs.yml

```diff
@@ -32,3 +32,54 @@ test:apidocs:
     - swagger-cli validate docs/*.yml
     # Verify that the Swagger docs follow the autodeployment requirements
     - python3 verify_docs.py `find docs -name "*.yml"`
+
+trigger:apidocs:rebuild-mender-api-docs:
```

Comment:
I think we could extend a bit the documentation above:

"This gitlab-ci template validates Swagger API specification and triggers Mender API Site rebuilds"